certora/specs/AccessControlDefaultAdminRules.spec

```diff
@@ -0,0 +1,484 @@
+  require timeSanity(e);
+  requireInvariant defaultAdminConsistency(defaultAdmin());
+  requireInvariant singleDefaultAdmin(e.msg.sender, defaultAdmin());
+
+  beginDefaultAdminTransfer@withrevert(e, newAdmin);
+  bool success = !lastReverted;
+
+  // liveness
+  assert success <=> e.msg.sender == defaultAdmin(), 
+    "only the current default admin can begin a transfer";
+
+  // effect
+  assert success => pendingDefaultAdmin_(e) == newAdmin, 
+    "pending default admin is set";
+  assert success => pendingDefaultAdminSchedule_(e) == e.block.timestamp + defaultAdminDelay(e), 
+    "pending default admin delay is set";
+}
+
+/*
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ Rule: A default admin can't change in less than the applied schedule                                                │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+*/
+rule pendingDefaultAdminDelayEnforced(env e1, env e2, method f, calldataarg args, address newAdmin) {
+  require intervalSanity(e1, e2);
+
+  uint48 delayBefore = defaultAdminDelay(e1);
+  address adminBefore = defaultAdmin();
+  beginDefaultAdminTransfer(e1, newAdmin);
+  f(e1, args);
+  address adminAfter = defaultAdmin();
+
+  assert adminAfter == newAdmin => ((e2.block.timestamp >= e1.block.timestamp + delayBefore) || adminBefore == newAdmin), 
+    "A delay can't change in less than applied schedule";
+}
+
+/*
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ Rule: pending default admin and it schedule can only change together                                                │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+*/
+rule pendingAdminAndScheduleCoupling(env e, address newAdmin) {
+  requireInvariant defaultAdminConsistency(defaultAdmin());
+  requireInvariant singleDefaultAdmin(e.msg.sender, defaultAdmin());
+
+  address pendingAdminBefore = pendingDefaultAdmin_(e);
+  uint48 scheduleBefore = pendingDefaultAdminSchedule_(e);
+
+  beginDefaultAdminTransfer(e, newAdmin);
+
+  assert (
+    scheduleBefore != pendingDefaultAdminSchedule_(e) &&
+    pendingAdminBefore == pendingDefaultAdmin_(e)
+  ) => newAdmin == pendingAdminBefore, "pending admin stays the same if the new admin set is the same";
+  assert (
+    pendingAdminBefore != pendingDefaultAdmin_(e) &&
+    scheduleBefore == pendingDefaultAdminSchedule_(e)
+  ) => (
+    // Schedule doesn't change if:
+    // - The previous schedule is block.timestamp and the delay is 0
+    (e.block.timestamp == scheduleBefore && defaultAdminDelay(e) == 0) ||
+    // - The defaultAdminDelay was reduced to a value such that added to the block.timestamp is equal to previous schedule
+    e.block.timestamp + defaultAdminDelay(e) == scheduleBefore
```

Comment:
`e.block.timestamp == scheduleBefore && defaultAdminDelay(e) == 0` 
is a special case of
`e.block.timestamp + defaultAdminDelay(e) == scheduleBefore`

so you can remove the first condition since the second part will cover it